pure3.md

```diff
@@ -0,0 +1,282 @@
+with no sufficiently capable ECUs.
+
+## Motivation
+
+Software supply chain attacks are on the rise with the
+[2022 State of the Software Supply Chain report](https://www.sonatype.com/state-of-the-software-supply-chain/introduction),
+noting a 742% increase over the course of the last three years. But, it was the
+[2020 Sunburst attack](https://www.cynet.com/attack-techniques-hands-on/sunburst-backdoor-c2-communication-protocol/)
+that really made both government and corporate CISOs take action. Affecting more
+than 100 companies and nine government agencies, the cause of the attack was
+Sunburst malware that had been uploaded via a routine update from Orion software
+produced by SolarWinds. The attack taught both public and private concerns some
+hard lessons, such as:
+
+* The very thing designed to protect the security of software-patches and
+  updates that fix routine vulnerabilities-can also be used to deliver malware
+  that can damage or corrupt.
+* As software products contain third-party components, program developers are
+  not in complete control of the contents of their products.
+* Malware can linger undetected for extended periods of time, allowing it to
+  infect every system that touches it. The Sunburst malicious code was part of
+  an update released by SolarWinds in the spring of 2020, yet it was not
+  reported until December of that year.
+
+Automotive software supply chains are particularly tempting targets for
+attackers. This is because automotive software runs on a large number of
+vehicles and updating all of them to recover quickly from an attack is a
+slow and difficult proposition. As such, a successful software supply chain
+attack means that thousands (or more) of vehicles are impacted. Any attack that
+targets a large number of vehicles on roads can do immeasurable damage. Further,
+while Uptane can securely deliver updates to vehicles to recover from the
+attack, ensuring every vehicle on the road receives the update immediately is
+very difficult given the circumstances of use of these vehicles. Therefore,
+strategies that can secure automotive software supply chains are vital to
+_avoid_ such attacks altogether.
+
+Governments around the world have recognized this fact. Regulations are
+forthcoming from the United Nations and other governing bodies, and standards
+have been published that specify how software running on automobiles must be
+secured. President Biden of the United States of America issued an Executive
+Order describing the need to improve the nation's cybersecurity. This EO, in
+particular, pushes for solutions like Software Bills of Materials (SBOMs) that
+list the components of software artifacts. This information is vital in ensuring
+developers are aware of the components in their supply chain so that they can
+fix vulnerabilities as and when they are discovered.
+
+Unfortunately, SBOMs do not provide end-to-end software supply chain security.
```

Comment:
Should clarify this more.  Perhaps use some details from the Verifiable SBOMs document as a basis for attacks that are not covered...

Reply:
I added a bit more but steered clear of diving into the possible incompleteness of SBOMs that also arises in the v-sboms work.